.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -244,6 +244,8 @@ swap:
   pseudo ops|pseudoops: pseudo-ops
   pull-down: pulldown
   recognise: recognize
+  Red Hat OpenJDK: Red Hat build of OpenJDK
```

Comment:
I see RHOJDK is also disallowed in the SSG PR. Is it something to include as well here? Or no worries if not.

Reply:
@rohennes - Good catch. Yes, that would make perfect sense  +1. Thank you :green_heart: